numba/targets/randomimpl.py

```diff
@@ -1153,7 +1153,8 @@ def shuffle_impl(context, builder, sig, args):
 
 @lower("np.random.shuffle", types.Buffer)
 def shuffle_impl(context, builder, sig, args):
-    res = _shuffle_impl(context, builder, sig, args, np.random.randint)
+    fn = _shuffle_impl if sig.args[0].ndim == 1 else _shuffle_impl_multidim
```

Comment:
There should probably be a direct test for this from `np.random.shuffle`?

Reply:
Done.